e2e/test_array_columns.py

```diff
@@ -0,0 +1,91 @@
+from playwright.sync_api import Page
+
+from piccolo_admin.example import ArrayColumns
+
+from .pages import AddRowPage, LoginPage
+
+
+def test_add_array_columns(page: Page, dev_server):
+    """
+    Make sure we can submit a form containing array columns, and it gets
+    saved in the database.
+    """
+    login_page = LoginPage(page=page)
+    login_page.reset()
+    login_page.login()
+
+    test_page = AddRowPage(page=page, tablename=ArrayColumns._meta.tablename)
+    test_page.reset()
+
+    test_page.add_array_value(
+        field="varchar",
+        value="Alice",
+    )
+
+    test_page.add_array_value(
+        field="integer",
+        value="1",
+    )
+
+    test_page.add_array_value(
+        field="email",
+        value="test@gmail.com",
+    )
+
+    test_page.submit_form()
+
+    response = ArrayColumns.select(
+        ArrayColumns.varchar,
+        ArrayColumns.integer,
+        ArrayColumns.email,
+    ).run_sync()
+
+    assert {
+        "varchar": ["Alice"],
+        "integer": [1],
+        "email": ["test@gmail.com"],
+    } in response
+
+
+def _test_value(page: Page, field: str, value: str):
+    login_page = LoginPage(page=page)
+    login_page.reset()
+    login_page.login()
+
+    test_page = AddRowPage(page=page, tablename=ArrayColumns._meta.tablename)
+    test_page.reset()
+
```

Comment:
It looks like we need to wait for Vue to load the page. It works for me locally. Try to add this
```python
# Let Vue JS finish loading
page.wait_for_timeout(1000)
```
like in `choices` tests.